lib/shared/widgets/widgets.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_multi_app/utils/colors.dart';
+
+const divider = Divider(
+  color: CustomColors.brownLight,
+);
+const progressIndicatorLight = SizedBox(
+  width: 24,
+  height: 24,
+  child: CircularProgressIndicator(
+    valueColor: AlwaysStoppedAnimation<Color>(Colors.white),
+    strokeWidth: 2,
+  ),
+);
+
+const progressIndicatorDark = SizedBox(
+  width: 24,
+  height: 24,
+  child: CircularProgressIndicator(
+    valueColor: AlwaysStoppedAnimation<Color>(Colors.brown),
+    strokeWidth: 2,
+  ),
+);
```

Comment:
No reason to create 2 different widgets in this case for only set another color. Make it one widget

Reply:
done